apps/studio/pages/project/[ref]/database/tables/index.tsx

```diff
@@ -24,24 +27,35 @@ const DatabaseTables: NextPageWithLayout = () => {
             <TableList
               onAddTable={snap.onAddTable}
               onEditTable={(table) => {
-                setSelectedTableToEdit(table)
+                setSelectedTableToEdit(postgresTableToEntity(table))
                 snap.onEditTable()
               }}
               onDeleteTable={(table) => {
-                setSelectedTableToEdit(table)
+                setSelectedTableToEdit(postgresTableToEntity(table))
                 snap.onDeleteTable()
               }}
               onDuplicateTable={(table) => {
-                setSelectedTableToEdit(table)
+                setSelectedTableToEdit(postgresTableToEntity(table))
                 snap.onDuplicateTable()
               }}
             />
           </div>
         </ScaffoldSection>
       </ScaffoldContainer>
 
-      <DeleteConfirmationDialogs selectedTable={selectedTableToEdit} />
-      <SidePanelEditor includeColumns selectedTable={selectedTableToEdit} />
+      {projectRef !== undefined &&
+        selectedTableToEdit !== undefined &&
+        isTableLike(selectedTableToEdit) && (
```

Comment:
because of these checks - clicking new table on the database/tables page doesnt open the side panel editor

Reply:
addressed